io/net/ethtool_test.py

```diff
@@ -131,14 +133,31 @@ def test_ethtool(self):
         for state, status in zip(["down", "up"], ["no", "yes"]):
             if not self.interface_state_change(self.iface, state, status):
                 self.fail("interface %s failed" % state)
-            cmd = "ethtool %s %s %s" % (self.args, self.iface, self.elapse)
-            ret = process.run(cmd, shell=True, verbose=True,
-                              ignore_status=True)
-            if ret.exit_status != 0:
-                if "Operation not supported" in ret.stderr_text:
-                    self.log.warn("%s failed" % self.args)
-                else:
-                    self.fail("failed")
+            if self.args == "-L":
+                cmd1 = "ethtool %s %s tx %s rx %s" % (self.args, self.iface, self.tx, self.rx)
```

Comment:
you can modify as below @TasmiyaNalatwad 
cmd1 = "ethtool %s %s tx %s rx %s" % (self.args, self.iface,
                                                                 self.tx, self.rx)

OR
you can use concatenation of strings like below
cmd1 = "ethtool %s %s tx " % (self.args, self.iface)
cmd1 = cmd1 + ' ' + "%s rx %s" % (self.tx, self.rx) 

which ever you feel easy, you can use that. we cannot ignore framework indentation style.

Reply:
Modified as per review comments. Framework Indentation style is satisfied and also executed pycodestyle which is not giving any indentation error now.